### ISSUE TITLE
HA class warning

### ISSUE THREAD

**Issue by MalcolmSpencer** [oziee/ha-solcast-solar#276]
### The problem

The following warning is in my HA Log on restart

Entity sensor.solcast_pv_forecast_forecast_today (<class 'custom_components.solcast_solar.sensor.SolcastSensor'>) is using state class 'measurement' which is impossible considering device class ('energy') it is using; expected None or one of 'total_increasing', 'total';

### What version of Solcast Integration has the issue?

v4.0.16

### What version of Home Assistant are you using?

2024.4.4

### Logs debug/info/error output

```txt
Entity sensor.solcast_pv_forecast_forecast_today (<class 'custom_components.solcast_solar.sensor.SolcastSensor'>) is using state class 'measurement' which is impossible considering device class ('energy') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/oziee/ha-solcast-solar/issues
```

**Comment by lazy-pete** [comment]
Same here. 

Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:575
integration: Sensor (documentation, issues)
First occurred: 22:09:19 (1 occurrences)
Last logged: 22:09:19

Entity sensor.solcast_pv_forecast_forecast_today (<class 'custom_components.solcast_solar.sensor.SolcastSensor'>) is using state class 'measurement' which is impossible considering device class ('energy') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/oziee/ha-solcast-solar/issues

**Comment by oziee** [comment]
Use search before posting to make sure it has not already been issued 

Repeat

**Comment by Antipodn** [comment]
Hi Oziee, I know you've closed this because its been mentioned before but you've never mentioned a possible fix. Yes, this is only an warning.  However, it clearly states that the sensor has been given the wrong State class or Device class according to HA standards. Can you please clarify your view. Thanks.

**Comment by oziee** [comment]
> Hi Oziee, I know you've closed this because its been mentioned before but you've never mentioned a possible fix. Yes, this is only an warning.  However, it clearly states that the sensor has been given the wrong State class or Device class according to HA standards. Can you please clarify your view. Thanks.

As mentioned in other places, this log message is nothing. Nothing in the integration is affected, it's just a warning not an error

Reading the "changelog" I see that this was addressed in v4.0.20
